ibu-imager/seedcreator/seedcreator.go

```diff
@@ -500,19 +495,39 @@ func (s *SeedCreator) waitTillOvnKubeNodeIsDown(ctx context.Context) {
 	defer deadlineCancel()
 }
 
-func (s *SeedCreator) getCatalogImages(ctx context.Context) ([]string, error) {
+// Catalog sources have pull always policy and there is no point to precache them.
```

Comment:
Please follow go function doc conventions

e.g. `filterCatalogImages filters a list of images... so that...`